tools/Raven.Debug/LogTrafficWatch/TrafficWatchReplay.cs

```diff
@@ -0,0 +1,313 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Net.Http;
+using System.Runtime.CompilerServices;
+using System.Security.Cryptography.X509Certificates;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using System.Text.RegularExpressions;
+using System.Threading;
+using System.Threading.Channels;
+using System.Threading.Tasks;
+using Raven.Client.Documents.Changes;
+
+namespace Raven.Debug.LogTrafficWatch
+{
+    public class TrafficWatchReplay : IDisposable
+    {
+        private readonly string[] _trafficFiles;
+        private readonly string _schema;
+        private readonly string _host;
+        private readonly int _port;
+        private readonly int _threads;
+        
+        public static HttpClient HttpClient;
+        private static readonly Channel<TrafficWatchChange> TrafficChannel = Channel.CreateBounded<TrafficWatchChange>(1024);
```

Comment:
Not very familiar with the rvn tool, but wonder if it is okay for those to be `static`

Reply:
System.Threading.Channels.Channel provides static methods for creating channels. So, it's okay to be static.

Reply:
Can you break the replay in the middle and start it over in the same rvn process? or can you do multiple replays in the same process?

If you can we have those issues:
1. You might have things in the queue from previous run
2. The Write might be completed for this channel, which will make it unusable.

Reply:
Channel and HttpClient are non-static now.